api_testsuite.go

```diff
@@ -126,6 +126,11 @@ type TestContainer struct {
 func (ats *Suite) parseAndRunTest(v interface{}, manifestDir, testFilePath string, k int, runParallel bool, r *report.ReportElement) bool {
 	//Init variables
 	loader := template.NewLoader(ats.datastore)
+	if externalHTTPServer != "" {
+		loader.HTTPServer = externalHTTPServer
+	} else if ats.httpServer.Addr != "" {
+		loader.HTTPServer = "//"+ats.HttpServer.Addr
+	}
```

Comment:
Store it in the "suite". Why have this code duplicated.

Also, why are you not assigning this to a url.URL ? That way you can complain if it does not parse.